src/convnets/densenet.jl

```diff
@@ -59,24 +64,24 @@ Create a DenseNet model
 - `nclasses`: the number of output classes
 """
 function densenet(inplanes, growth_rates; reduction = 0.5, nclasses = 1000)
-  layers = []
-  append!(layers, conv_bn((7, 7), 3, inplanes; stride = 2, pad = (3, 3), bias = false))
-  push!(layers, MaxPool((3, 3), stride = 2, pad = (1, 1)))
-
-  outplanes = 0
-  for (i, rates) in enumerate(growth_rates)
-    outplanes = inplanes + sum(rates)
-    append!(layers, dense_block(inplanes, rates))
-    (i != length(growth_rates)) &&
-      push!(layers, transition(outplanes, floor(Int, outplanes * reduction)))
-    inplanes = floor(Int, outplanes * reduction)
-  end
-  push!(layers, BatchNorm(outplanes, relu))
-
-  return Chain(Chain(layers),
-               Chain(AdaptiveMeanPool((1, 1)),
-                     MLUtils.flatten,
-                     Dense(outplanes, nclasses)))
+    layers = []
+    append!(layers, conv_bn((7, 7), 3, inplanes; stride = 2, pad = (3, 3), bias = false))
+    push!(layers, MaxPool((3, 3), stride = 2, pad = (1, 1)))
+
+    outplanes = 0
+    for (i, rates) in enumerate(growth_rates)
+        outplanes = inplanes + sum(rates)
+        append!(layers, dense_block(inplanes, rates))
+        (i != length(growth_rates)) &&
+            push!(layers, transition(outplanes, floor(Int, outplanes * reduction)))
+        inplanes = floor(Int, outplanes * reduction)
+    end
+    push!(layers, BatchNorm(outplanes, relu))
+
+    return Chain(Chain(layers),
+                 Chain(AdaptiveMeanPool((1, 1)),
+                       MLUtils.flatten,
+                       Dense(outplanes, nclasses)))
```

Comment:
This and other calls appear to conflict with the advice "Function calls which cannot fit on a single line within the line limit should be broken up such that the lines containing the opening and closing brackets are indented to the same level while the parameters of the function are indented one level further..." under https://github.com/SciML/SciMLStyle#whitespace

Reply:
I don't quite get that bullet. The guideline and the example below it seem in direct opposition?

Reply:
I could've sworn I saw an example like:
```julia
long_function_name(
    long_arg_1_which_requires_a_break,
    long_arg_2_which_requires_a_break,
    ...
)
```
But looking back I cannot find one. Had I looked at other style guides recently this could've been an easy mixup, but I have not. Very confused now...

Reply:
I'm personally a fan of the standard Julia convention which the example and the formatter seem to follow.